pkg/plugin/plugin.go

```diff
@@ -114,6 +116,125 @@ func generateRandomMac() net.HardwareAddr {
 	return net.HardwareAddr(append(prefix, suffix...))
 }
 
+func setupSriovInterface(contNetns ns.NetNS, ifName string, mtu int, pciAddrs string) (*current.Interface, *current.Interface, error) {
+	hostIface := &current.Interface{}
+	contIface := &current.Interface{}
+
+	// 1. get VF netdevice from PCI
+	vfNetdevices, err := sriovnet.GetNetDevicesFromPci(pciAddrs)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// Make sure we have 1 netdevice per pci address
+	if len(vfNetdevices) != 1 {
+		return nil, nil, fmt.Errorf("failed to get one netdevice interface per %s", pciAddrs)
+	}
+	vfNetdevice := vfNetdevices[0]
+
+	// 2. get Uplink netdevice
```

Comment:
Could you please explain in a comment what is uplink representor?

Reply:
done